deploy/olm-catalog/bundle/manifests/migration.openshift.io_migplans.yaml

```diff
@@ -50,6 +51,7 @@ spec:
           description: MigPlanSpec defines the desired state of MigPlan
           properties:
             closed:
+              description: If the migration was successful for a migplan, controller sets this value to True indicating that after one successful migration no new migrations can be carried out for this migplan.
```

Comment:
This is not accurate, this field can only be set manually, it will never get set automatically upon plan completion

Reply:
Updated the description.